### DISCUSSION TITLE
How can bromite's development be sped up in a meaningful way?

### DISCUSSION THREAD

**Discussion by peternrdstrm** [bromite/bromite#2441] [General]
Bromite as in features is the beat android privacy browser. 

It's only issue is updates. 
So I ask you @csagan5 and @uazo , how can we the community help to speed this development up? 
I'd be happy to beta test, and such, and I'm sure many others would too.

**Comment by uazo** [comment]
> I'd be happy to beta test

explain yourself. what do you mean by being a beta tester? what are your skills?

**Reply by peternrdstrm** [reply]
I have some prior experience beta-testing stuff, got a new gen pixel phone for a good testing device and I'm generally familiar with bugreports and such, I've read crash logs a lot. Also, I understand programming stuff, so I could deal with panics or crashes and stuff

**Reply by uazo** [reply]
OK, I get it.

what bromite would need is not so much a beta tester as you describe, after all, crashes are public and you are welcome to try your hand at it, rather than a beta tester who would work out the steps needed to verify the changes that are made.
a sort of checklist, with the future aim of verifying how to automate it.
in my opinion that could be useful.

**Reply by csagan5** [reply]
At some point it would be beneficial to automate this checklist with one of these services that allow using multiple devices with multiple Android versions. We do not need to cover the official Chromium testsuites but a good start would be to cover the basic UI flows.

**Comment by csagan5** [comment]
> It's only issue is updates.

No. The issues are here: https://github.com/bromite/issues

> how can we the community help to speed this development up?

If you can contribute some patch to fix a bug or implement a feature, that will speed development up. However I believe that by "development speed" you mean "release speed", which is not the same thing.

**Reply by peternrdstrm** [reply]
> > It's only issue is updates.
> 
> No. The issues are here: https://github.com/bromite/issues
> 
> > how can we the community help to speed this development up?
> 
> If you can contribute some patch to fix a bug or implement a feature, that will speed development up. However I believe that by "development speed" you mean "release speed", which is not the same thing.

yes, I mean that. I might try bugfixes, but I've never submited to projects before
